lib/levelup.js

```diff
@@ -65,9 +65,17 @@ function LevelUP (location, options, callback) {
     options  = {}
   }
```

Comment:
@juliangruber I'm gonna do it your way here and start off by moving db to `options.db` and once that is done get rid of `location`